containers/deploy/service-accounts/tasks/main.yml

```diff
@@ -2,26 +2,18 @@
 - name: Login as system admin
   command: oc login -u system:admin
   when: minishift or cluster_up
-  tags:
-    - start
 
 - name: Add anyuid scc to anyuid service account
   command: oc adm policy add-scc-to-user anyuid system:serviceaccount:{{ project_name }}:anyuid
-  tags:
-    - start
 
 - name: Login as developer
   command: oc login -u developer -p a
   when: minishift or cluster_up
-  tags:
-    - start
 
 - name: Ensure on project
```

Comment:
This should no longer be necessary, the namespace field in the k8s_raw objects will take precedence

Reply:
* assuming there are no oc commands being run